x/ccv/provider/keeper/distribution.go

```diff
@@ -240,6 +254,12 @@ func (k Keeper) GetConsumerRewardsPool(ctx sdk.Context) sdk.Coins {
 func (k Keeper) ComputeConsumerTotalVotingPower(ctx sdk.Context, chainID string) (totalPower int64) {
 	// sum the consumer validators set voting powers
 	for _, v := range k.GetConsumerValSet(ctx, chainID) {
+
+		// only consider the voting power of a validator that would receive rewards (i.e., validator has been validating for a number of blocks)
+		if !k.IsEligibleForConsumerRewards(ctx, v.Height) {
```

Comment:
What happens here when a new consumer chain is added? Are all the validators ineligible? If so, is it safe for ComputeConsumerTotalVotingPower to return zero? Also, what happens with the rewads for the first warmup period?

Reply:
See above in the PR description:
>Finally, note that in a consumer chain that just starts, no validator would receive rewards during the first NumberOfEpochsToStartReceivingRewards epochs. During this time, any rewards sent to the provider from the consumer chain through IBC transfers will be allocated to the community pool.

If no validator is eligible, then rewards would [go to the community pool](https://github.com/cosmos/interchain-security/blob/864b3acd5a26842049ec23e6f3bcbe0d4ab78519/x/ccv/provider/keeper/distribution.go#L104). The same applies for the first warmup period. No validator would get rewards and the rewards would end up in the community pool.